lesson_1/convertBytesToHuman.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * В этом задании надо разработать функцию
+ * `convertBytesToHuman`. Эта функция  должна принимать
+ * аргумент `bytes` только числового типа.
+ * На выходе функция должна отдать
+ * человекопонятную строку, которая будет
+ * отражать размер файла. Примеры использования:
+ * `convertBytesToHuman(1024) === '1 KB';`
+ * `convertBytesToHuman(123123123) === '117.42 MB';`
+ * Необходимо предусмотреть защиту от
+ * передачи аргументов неправильного типа
+ * и класса (например, отрицательные числа)
+ */
+
+export default function convertBytesToHuman(bytes) {
+  if (bytes === 0) return '0 B'
+  if (isNaN(bytes) || typeof bytes !== 'number' || bytes < 0) return false;
+
+  const humanSizes = ['B', 'KB', 'MB', 'GB', 'TB'];
+  const sizeIndex = Math.min(humanSizes.length - 1, Math.floor(Math.log(bytes) / Math.log(1024)));
+  const value = (bytes / 1024**sizeIndex);
```

Comment:
По стилю кода, любой (не унарный) арифметический оператор отделяется пробелами с обоих сторон. Возможно, здесь отсутствие пробелов вызвано желанием явно выполнить возведение в степень вперёд деления. Впрочем, по [приоритету операторов](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Operators/Operator_precedence), возведение в степень и так выше.

```suggestion
  const value = (bytes / 1024 ** sizeIndex);
```